soc/arm/st_stm32/stm32g0/Kconfig.defconfig.stm32g071xx

```
@@ -12,4 +12,12 @@ config SOC
 config NUM_IRQS
 	default 32
 
+config CLOCK_STM32_PLL_Q_DIVISOR
+	int "PLL Q Divisor"
+	depends on CLOCK_STM32_SYSCLK_SRC_PLL
+	default 2
+	range 2 8
+	help
+	  PLL Q VCO divisor, allowed values: 2-8
+
```

If some modification has to be done on clock_control driver, please split in a different commit.
Then, please keep Kconfig symbols in their own driver when doable.
In this case, this can be done as follows in `drivers/clock_control/Kconfig.stm32g0`:
```suggestion
config CLOCK_STM32_PLL_Q_DIVISOR
	int "PLL Q Divisor"
	depends on CLOCK_STM32_SYSCLK_SRC_PLL && (SOC_STM32G031XX || SOC_STM32G071XX)
	default 2
	range 2 8
	help
	  PLL Q VCO divisor, allowed values: 2-8
          Limited to STM32G0X1 variants
```